portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase

```diff
@@ -185,6 +185,226 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-10698 | Automation ID: LRAC-11119 | Test Summary: Empty state of no assets with data source and property"
+	@ignore = "true"
+	@priority = "2"
+	test CheckAllMessagesWhenHasPropertyAndDataSourceButNoData {
+		property test.name.skip.portal.instance = "EmptyState#CheckAllMessagesWhenHasPropertyAndDataSourceButNoData";
+
+		// AC Refactor: Depends on bug fix (LRAC-11153)
+
+		task ("Close the onboarding modal") {
+			ACUtils.closeModal();
+		}
+
+		task ("Copy the token from AC") {
+			ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Connect DXP to AC no data") {
+			ACUtils.launchDXP();
+
+			ACDXPSettings.goToInstanceSettingsAC();
+
+			ACDXPSettings.connectAnalyticsCloud();
+		}
+
+		task ("Go to AC workspace") {
+			ACUtils.launchAC(workspaceName = "New AC Workspace");
+		}
+
+		task ("Go to Assets") {
+			ACNavigation.goToAssets();
+		}
+
+		task ("Go to Blogs, check empty state message") {
+			ACUtils.viewGenericText(textValueList = "There are no visitors data found.,Check back later to verify if data has been received from your data sources, or you can try a different date range.,Learn more about blogs.");
+
+			ACUtils.clickGenericHyperlink(hyperlinkText = "Learn more about blogs.");
+
+			SelectWindow(locator1 = "title=Blogs — Liferay Learn");
+
+			ACUtils.viewGenericText(textValueList = "Documentation,Blogs");
+
+			Close.closeWindow();
+		}
+
+		task ("Go to Documents and Media Tab, check empty state message") {
+			ACNavigation.switchTab(tabName = "Documents and Media");
+
+			ACUtils.viewGenericText(textValueList = "There are no visitors data found.,Check back later to verify if data has been received from your data sources, or you can try a different date range.,Learn more about documents and media.");
+
+			ACUtils.clickGenericHyperlink(hyperlinkText = "Learn more about documents and media.");
+
+			SelectWindow(locator1 = "title=Documents and Media — Liferay Learn");
+
+			ACUtils.viewGenericText(textValueList = "Documentation,Documents and Media");
+
+			Close.closeWindow();
+		}
+
+		task ("Go to Forms Tab, check empty state message") {
+			ACNavigation.switchTab(tabName = "Forms");
+
+			ACUtils.viewGenericText(textValueList = "There are no visitors data found.,Check back later to verify if data has been received from your data sources, or you can try a different date range.,Learn more about forms.");
+
+			ACUtils.clickGenericHyperlink(hyperlinkText = "Learn more about documents and media.");
```

Comment:
Hey @joshitagaki, LGTM for automating this test. Only one minor adjustment, I think the hyperlink is "Learn more about forms." yes? After this fix, your code is very good for me.